### PR TITLE
Fix CapcomID persistence across session

### PR DESCRIPTION
**Pull request by sepalani**
This PR prevents multiple users to use simultaneously the same Capcom ID. It also provides a wrapper that can forward tiny error messages to the game.

Ready to be reviewed & tested.

### CODE REVIEW

File: mh/session.py [hunk 1]
```diff
@@ -65,15 +65,33 @@ def get(self, connection_data):
             self.online_support_code = to_str(
                 pati.unpack_string(connection_data.online_support_code)
             )
-        return DB.get_session(self)
+        session = DB.get_session(self)
+        if session != self:
```
Comment (InusualZ) [inline mh/session.py:69]: This code is a little bit confusing, you are inside a class named `Session` and asking a database about getting session... Once I looked a bit thorough, I understand what was happening. Maybe giving a new name to `DB.get_sesssion()`?

**Review by InusualZ — APPROVED** [review]
Just one comment, other than that code LGTM

File: mh/session.py (hunk 1 above)
Comment (sepalani) [reply mh/session.py:69]: I think finding a new name will be hard for that case _(sessions which aren't connected yet)_. What about changing its parameter instead?

Would something like this sounds fine to you:
```
session = DB.get_session(pat_ticket) or self
```
Or alternatively:
```
session = DB.get_session(pat_ticket, default=self)
```

Comment (InusualZ) [reply mh/session.py:69]: I think the first option give a little more about the intent of the function.

Comment (sepalani) [reply mh/session.py:69]: Done in the last commit.

**Comment by InusualZ** [comment]
LGTM

**Pull request merged**